radiography/src/main/java/radiography/ViewStateRenderer.kt

```diff
@@ -9,23 +11,33 @@ package radiography
  *  }
  * ```
  */
-class ViewStateRenderer<in T> @PublishedApi internal constructor(
-  private val renderedClass: Class<T>,
-  private val renderer: AttributeAppendable.(T) -> Unit
+abstract class ViewStateRenderer<in T> internal constructor(
```

Comment:
@pyricau I kept this constructor internal, but I'm not sure why this class is so locked down. Why not make this an interface and allow direct implementations? I wanted to be able to override it to make a custom implementation for compose support that I could specifically search for by type, but I could also just make the renderedType property internal and search by that if you really want this to stay final.

Reply:
More complete thoughts in this comment: https://github.com/square/radiography/pull/43#issuecomment-674167587